### DISCUSSION TITLE
One of the forms I submitted was approved but I did not receive any benefit.

### DISCUSSION THREAD

**Discussion by BhathikaGimhan** [community/community#62210] [GitHub Education]
![Screenshot_2023-07-30-17-07-24-53_e307a3f9df9f380ebaf106e1dc980bb6.jpg](https://github.com/community/community/assets/74421641/1777b01e-2be5-422a-b47f-d64a02c2ac64)

My email: e2145199@bit.uom.lk

This is my educational email from my university. I gave my email. Even after uploading a screenshot on my portal, I got this email from github. Please explain to me why

**Comment by priyankarpal** [comment]
use the mail that you have used to create the github account

**Comment by BhathikaGimhan** [comment]
Thanks for your reply
